### PR TITLE
Enhancements/user avatar

### PR DESCRIPTION
**Pull request by BandanaPandey**
Added avatar in the users table to store user profile image

**Changes:**
- Added migrations to have avatar in the users table
- Incorporated avatar in the User Update API

### CODE REVIEW

File: apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex [hunk 1]
```diff
@@ -182,6 +183,18 @@ defmodule AcqdatApiWeb.UserController do
...
+
+        params =
+          case is_nil(params["image"]) do
+            true ->
+              params
+
+            false ->
+              add_image_url(conn, params)
+          end
+
```
Comment (arjun289) [inline apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex:197]: please do not use update function here, make use of `defdelegate`. Also,  for updating params create a separate function. The function is too big.

Comment (BandanaPandey) [reply apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex:197]: @arjun289 I didn't get it update function, which you are referring ?
Are you referring to this **UserModel.update()** ?

Comment (BandanaPandey) [reply apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex:189]: Did the changes

File: apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex [hunk 2]
```diff
@@ -3,9 +3,12 @@ defmodule AcqdatApiWeb.UserController do
   alias AcqdatApi.User
   alias AcqdatCore.Model.User, as: UserModel
   alias AcqdatApi.ElasticSearch
+  alias AcqdatApi.Image
   import AcqdatApiWeb.Validators.User
   import AcqdatApiWeb.Helpers
 
+  defdelegate update_user(user, params), to: UserModel
```
Comment (arjun289) [inline apps/acqdat_api/lib/acqdat_api_web/controllers/user_controller.ex:10]: please move this to `AcqdatApi.User`

**Pull request merged**